app/web/src/features/messages/utils.ts

```diff
@@ -58,3 +58,18 @@ export function groupChatTitleText(
         .map((user) => firstName(user?.name))
         .join(", ");
 }
+
+export function groupChatUsernameText(
```

Comment:
We don't need JSDoc comments as often when using typescript, but it's not clear what this function does just from the name and type. By making this change, hovering the mouse over the function name in VSCode (or any good IDE) will have the written description so you don't have to switch files and read the funciton carefully to figure out what's going on.
```suggestion
/** Returns the other user's username, or null if there are more than 2 users. */
export function groupChatUsernameText(
```

Reply:
Maybe also worth renaming this function to `groupChatOtherUsername`

Reply:
Or maybe something like `formatGroupChatUsernames`? Not too strongly opinionated, but often when you have to write a comment to explain a function (especially when it's not some unavoidable obscure code that has to be done for perf reasons or something), it's a bit of a code smell

Reply:
Actually, I'm confused why we need this at all. Doesn't  `groupChatTitleText` already format the names correctly? Surely we could bundle the logic in there and instead of running the same check/processing twice?

Reply:
groupChatTitleText format the name Good but it does not takes the username used for the url /user/{username} ? Or maybe i can use the same function and add a parameters to the function like `returnUsername` as boolean ? @darrenvong